### PR TITLE
Generating and activating Dots properly

### LINKED ISSUE #36: Dots is not working properly when there is more than 1 item per page
Dots should work similarly as pagination. eg:

If we have 6 items, and we set perPage={2}, it should appear only 3 dots. However, is appearing 6 dots (one for each item), and this is causing a bug, as we can even see in the demo link in the READ.me: 

https://svelte.dev/repl/3953567d530b41d087ab7eaa8e7e632a?version=3.22.3

We can see we have 4 items and 4 dots. but we should have only 2 dots. 
4 / perPage(3 default) = Math.ceil(1.3) = 2.

We also can see that after dragging to right, the dots will remain without the class "active" until it gets to the item 0 again

### PR DESCRIPTION
**Pull request by johnyvelho**
Fixes #36

### CODE REVIEW

File: src/Carousel.svelte [hunk 1]
```diff
@@ -8,8 +8,8 @@
 	</div>
     {#if dots}
 	<ul>
-		{#each pips as pip, i}
-		<li on:click={() => go(i)} class={currentIndex === i ? "active" : ""}></li>
+		{#each Array(totalDots) as _, i}
```
Comment (antony) [inline src/Carousel.svelte:11]: try `{#each {length: totalDots} as i}` ;)

File: src/Carousel.svelte [hunk 2]
```diff
@@ -8,8 +8,8 @@
...
+		<li test={i} on:click={() => go(i*perPage)} class={isDotActive(currentIndex, i) ? "active" : ""}></li>
```
Comment (antony) [inline src/Carousel.svelte:12]: not sure you meant to leave `test` in there?

File: src/Carousel.svelte (hunk 1 above)
Comment (johnyvelho) [reply src/Carousel.svelte]: Thanks!

File: src/Carousel.svelte (hunk 2 above)
Comment (johnyvelho) [reply src/Carousel.svelte]: certainly not haha. thanks!

**Comment by antony** [comment]
Released as v2.7.0 :)

**Pull request merged**